src/meetings.ts

```diff
@@ -11,9 +11,9 @@ export interface IMeeting {
 export function formatMeeting({ day, room, times: [start, end] }: IMeeting): string {
     let formatted = `**Day**: ${day}` +
         `\n**Room**: ${room}` +
-        `\n**Starts**: ${start}`;
+        `\n**Starts**: ${moment(start).format("h:mm A")}`;
```

Comment:
I think this is already a moment so it probably isn't necessary to do `moment(start)`, instead just `start.format()`